index.bs

```diff
@@ -437,6 +446,9 @@ do support suborigins. The following discussions discuss the three major
 cross-origin mechanisms that are relevant: <a>CORS</a>, <a>`postMessage`</a>,
 and Workers [[!WORKERS]].
 
+Issue: TODO(devd): Making things specific to XHR or CORS is weird. We should
```

Comment:
Isn't that, more or less, what I wrote? Is it just perhaps the heading that you don't like? Should we rename it "Fetching" instead of "CORS"?